storybook/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -52,10 +52,8 @@ const defaultTabs = [
   ...(() =>
     tabsContent.map((content, index) => (
```

Comment:
Also out of scope, but: we can destructure `label` and `content` here as well. And we should  use `label` as the key, instead of an index.

Reply:
Fixed in https://github.com/Amsterdam/design-system/pull/1275.

We already used `label` as the key. The index is for that `tab` prop – although I’ve been wondering if there’s a better approach for it.